test/pom.xml

```
@@ -83,7 +83,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>1690.vfcb_959006c07</version>
+      <version>1692.v41c59b_73de66</version>
```

upstream build is broken due to artifactory... pushed a snapshot so try that for purposes of testing.
```suggestion
      <version>1692.v41c59b_73de66-SNAPSHOT</version>
```